gateway/1.5/modules/ROOT/pages/policies-pdk-develop-custom-policies.adoc

```diff
@@ -2,4 +2,48 @@
 ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
-:imagesdir: ../assets/images
\ No newline at end of file
+:imagesdir: ../assets/images
+
+After installing the xref:policies-pdk-prerequisites.adoc[], you can develop custom policies by using the Policy Development Kit (PDK).
+
+The custom policy lifecycle consists of the following docs that explain all steps of developing a custom policy that occur in PDK. The following docs do not include steps that occur outside of PDK, such as xref:policies-pdk-deploy-policies.adoc[applying a policy].
+
+After you xref:policies-pdk-create-project.adoc[create a new project], you might iteratively complete some steps. For example, while writing the policy's source code, you might need to return to xref:policies-pdk-create-schema-definition.adoc[add additional parameters], or you might need to recompile the policy after debugging the source code.
+
+The commands associated with each step are provided for reference. However, configuration is necessary before running some of the commands.
```

Comment:
I have the same questions as before: Do you need the first sentence? What do you mean by "configuration is necessary...?" Configuration of what?

Reply:
I thought I deleted this, sorry.